cocotb/drivers/ad9361.py

```diff
@@ -1,214 +0,0 @@
-# Created on Aug 24, 2014
-# 
-# @author: msnook
-
-import cocotb
-from cocotb.triggers import Timer, RisingEdge, Event
-from cocotb.drivers import BusDriver
-from cocotb.binary import BinaryValue, BinaryRepresentation
-
-from collections import deque
-
-
-class AD9361(BusDriver):

```

Comment:
Instead of deleting this class as part of this Pr, I'd just remove the base class, and open an issue about removing it.

Reply:
I'll leave the AS9361 in as is then and make separate issue for removing it.
The code will only cause problems when actually trying to create an object of that type and also when removing the base class.

Reply:
Does this even use a single method from `BusDriver`? If not, removing the base class ought to be perfectly safe